strings/tips.txt

```diff
@@ -153,18 +153,15 @@ As a Cultist, your team starts off very weak, but if necessary can quickly conve
 As a Cultist, the Blood Boil rune will deal massive amounts of brute damage to non-cultists, stamina damage to Ratvarian scum, and some damage to fellow cultists of Nar-Sie nearby, but will create a fire where the rune stands on use.
 As a Cultist, you can create an army of manifested goons using a combination of the Manifest rune, which creates homunculi from ghosts, and the Blood Drain rune, which drains life from anyone standing on any blood drain rune.
 As a Cultist or Servant, check the alert in the upper-right of your screen for all the details about your cult's current status and objective.
-As a Servant, your Clockwork Slab fits in pockets and does not need to be held to communicate with other Servants.
-As a Servant, remember that while the selection of scripture and tools Servant cyborgs get is limited, it is still extremely useful, and some of it is unique; for example, engineering and janitor cyborgs get a proselytizer that can use their own power in addition to its own.
-As a Servant, the Judicial Visor is an effective defensive combat tool in small spaces, as it immediately slows and briefly knocks down enemies hit while giving them incentive to avoid staying in the hit area.
```

Comment:
And this one

Reply:
I tried to replace the tips with actual points of note, instead of just describing what scripture does since people can just see that in the slab.

Reply:
I agree that the basic tips are stuff you could get from the slab, but stuff like "as it immediately slows and briefly knocks down enemies hit while giving them incentive to avoid staying in the hit area."/"Clockwork Floors will also rapidly heal toxin damage in Servants, allowing the Compromise more effectiveness."/"As a Servant, you can unwrench Clockwork Structures to move them around." all feel like they're hidden nuggets of info that most people would like